mozilla/blurts-server
# Start redirecting new users to the welcome screen

# References: 
Jira: MNTOR-2264
Figma: N/A




# Description

Here, "new users" are defined as those who are in the US, have not yet run a data broker scan, and did not create their Monitor account before the release of data broker scanning, as determined by the date set in the `BROKER_SCAN_RELEASE_DATE` environment variable.

# How to test

Set `BROKER_SCAN_RELEASE_DATE` to a date before you created your account, and make sure you haven't run a scan yet. If you open the dashboard, you should be redirected to the Welcome flow. But if your account is older than that date, you should just be able to look at the dashboard.

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, would involve lots of mocking
